data/remnant missions.txt

```diff
@@ -1252,3 +1252,237 @@ mission "Remnant: Expanded Horizons Quarg 3"
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Thank you. We are looking forward to seeing what comes from this."`
+				accept
+			label refuse
+			`	She taps something on her screen. "OK, I've notified them they will have to wait a bit. Let us know when you are ready."`
+				defer
+	on complete
+		conversation
+			`As you approach your designated pad you spot a team of researchers with unloading equipment hustling out of a nearby entrance. Once you have completed your shutdown sequence they quickly set to work unloading the drive. Once the task is complete, they pause for a moment to hand you <payment>. "Thanks for finding this," they sing in unison. "Our labs are eager to begin work on your most recent find, Captain <first>. Our thanks!"`
+			choice
+				`	"How is your research going?"`
+					goto studying
+				`	"Good luck with that."`
+					end
+			label studying
+			`	"Good, and about to get better," they sing. "We are working on various aspects of a theory that the jump drive actually disregards traditional jumping entirely, and uses the folding mechanism to put the ship itself into a pocket, while somehow triggering an opening to said pocket in the target system. If this is true, it could be fairly dangerous to try activating these anywhere close to fragile things like cities or mountains or whatnot, which is why they put our lab on the far side of the planet from all such things. So we have a bit of a trip to get to our lab, but it should be exciting once we get there. Fare well!" With that, they head off towards a large vehicle nearby.`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 5"
+	name "Broken Jump Drive Delivery"
+	description "A team of engineers on <planet> would like a broken jump drive to study."
+	to offer
+		has "Remnant: Broken Jump Drive 4: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`You stop by the director's desk to report having salvaged another broken jump drive for one of their research teams. The director nods in appreciation and checks his list. "There is a new team starting a project on <planet>. Could you take it there please?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Much appreciated. The sooner they get it, the sooner they can start on their project. That being said, I have spoken with the other directors, and we have decided that we are getting so many requests that we will put these into the job board for you. Any time the landing scans detect one of these broken jump drives, the system will display a message on the job board listing where the next one needs to be delivered."`
+				accept
+			label refuse
+			`	He looks surprised, "Really? Oh. Let us know when you are ready to make the delivery. The researchers will be waiting for it."`
+				defer
```

Comment:
if the player refuses, won't they get the same dialogue again?

Reply:
That's true, they would. Although with possibly a new destination.